README.md

```diff
@@ -1 +1,5 @@
-# project-rl
\ No newline at end of file
+# Project-reinforcement-learning
```

Comment:
Maybe keep the title same as repo name? `project-rl`

Reply:
this is more readable